### PR TITLE
Mariadb update

### PR DESCRIPTION
**Pull request by grooverdan**
Bulk update largely to cover the new names being available and a few other cleanups.

### CODE REVIEW

File: mariadb/content.md [hunk 1]
```diff
@@ -89,31 +89,39 @@ $ docker run -it --rm %%IMAGE%%:tag --verbose --help
...
+### `MARIADB_RANDOM_ROOT_PASSWORD` / `MYSQL_RANDOM_ROOT_PASSWORD`
 
-### `MYSQL_ALLOW_EMPTY_PASSWORD`
+Set to a non-empty value, like `yes`, to generate a random initial password for the root user. The generated root password will be printed to stdout (`GENERATED ROOT PASSWORD: .....`).
 
-This is an optional variable. Set to a non-empty value, like `yes`, to allow the container to be started with a blank password for the root user. *NOTE*: Setting this variable to `yes` is not recommended unless you really know what you are doing, since this will leave your MariaDB instance completely unprotected, allowing anyone to gain complete superuser access.
+### `MARIADB_ROOT_HOST` / `MYSQL_ROOT_HOST`
 
-### `MYSQL_RANDOM_ROOT_PASSWORD`
+This is hostname part of the root user created. By default this is `%`, however it can be set to any default [MariaDB allowed hostname component](https://mariadb.com/kb/en/create-user/#host-name-component).
```
Comment (tianon) [inline mariadb/content.md:112]: "This is hostname part" seems like it's missing a word (perhaps "This is _the_ hostname part" ?)

**Review by tianon — COMMENTED** [review]
Generally LGTM, minus one minor bit that seems like a typo/oversight :smile:

**Review by yosifkit — APPROVED** [review]
LGTM. Waiting on an update to the images that contains this functionality so that the documentation stays in sync.

**Comment by grooverdan** [comment]
Sure. I'm happy to see #1898 land and I'll rebase this before next expected release 2021-04-30

**Comment by yosifkit** [comment]
I see more commits here; now that an image is up with `MARIADB_*` variables, (https://github.com/docker-library/official-images/pull/10070), are these docs ready?

**Comment by grooverdan** [comment]
> I see more commits here; now that an image is up with `MARIADB_*` variables, ([docker-library/official-images#10070](https://github.com/docker-library/official-images/pull/10070)), are these docs ready?

The one that is up is the Alpha release. The rest are coming I expect within a week. Can we merge this when the next stable releases get updated please.

**Comment by grooverdan** [comment]
Thank you.

**Pull request merged**